src/components/WrapEtherBtn.tsx

```diff
@@ -222,14 +244,38 @@ const WrapUnwrapEtherBtn: React.FC<WrapUnwrapEtherBtnProps> = (props: WrapUnwrap
                   required
                   ref={register({
                     pattern: { value: validInputPattern, message: 'Invalid amount' },
-                    validate: { positive: validatePositiveConstructor('Invalid amount') },
+                    validate: {
+                      positive: validatePositiveConstructor('Invalid amount'),
+                      max: (value: string): string | true => {
+                        const amount = parseAmount(value, DEFAULT_PRECISION) || ZERO
+
+                        if (balance && amount.gt(balance)) {
+                          // Not enough balance
+                          return "The amount cannot be bigger than what's available"
+                        } else {
+                          // Enough balance
+                          return true
+                        }
+                      },
+                    },
                     required: 'The amount is required',
                     min: 0,
                   })}
                 />
               </InputBox>
             </div>
-            {amountError && <p className="error">Invalid amount</p>}
+            {amountError && <p className="error">{amountError.message}</p>}
+            {wrapAllBalance && (
```

Comment:
I wouldn't care about this use-case at all, but whatever

Reply:
It's not the core part of this, but it's just taking care of a edge case. I think it's better now that I removed the link for setting the max amount in the WRAPPING